# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnue-forms/fix_curses.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
gnue: trunk/gnue-forms/src/uidrivers/curses/UIdriver.py
Diff for /trunk/gnue-forms/src/uidrivers/curses/UIdriver.py between version 6851 and 7225
version 6851, Mon Jan 3 20:59:28 2005 UTC 	version 7225, Sat Mar 19 00:30:20 2005 UTC

--- gnue-forms/src/uidrivers/curses/UIdriver.py	2005/01/03 20:59:28	6851
+++ gnue-forms/src/uidrivers/curses/UIdriver.py	2005/03/19 00:30:20	7225
@@ -41,10 +41,10 @@
   # ---------------------------------------------------------------------------
 
   def __init__ (self, eventHandler, name = "Undefined", disableSplash = None,
-                parentContainer = None):
+                parentContainer = None, moduleName = None):
 
     GFUserInterfaceBase.__init__ (self, eventHandler, name, disableSplash,
-                                  parentContainer)
+                                  parentContainer, moduleName)
 
     self.__screen = curses.initscr ()
     atexit.register (curses.endwin)
